user-guide/Advanced_Functionality/Swarming/RollbackSwarming.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: SwarmingRollback
+---
+
+# Partially rolling back Swarming
+
+> [!IMPORTANT]
+> There is currently no good way to disable Swarming and keep everything in the DMS intact. The info below only partially recovers some data. If you are using a test system and it does not matter if all data are lost, a clean reinstall is advised.
+
+When Swarming is enabled, the element config XML files (e.g. `Element.xml`, `ElementData.xml`, `Description.xml`, etc.) are moved from disk to the database. There is currently no procedure to move them back to disk.
```

Comment:
Is it just these 3 XML files that get moved to the database, or are there others as well? If it's just these 3, I'd remove the "etc.":
```suggestion
When Swarming is enabled, the element config XML files (e.g. `Element.xml`, `ElementData.xml`, and `Description.xml`) are moved from the disk to the database. There is currently no procedure to move them back to the disk.
```

Reply:
There are also matrix discreets files (e.g. ports.xml). There's zero or more of these without fixed names (names as defined in the protocol -- https://docs.dataminer.services/develop/schemadoc/Protocol/Protocol.Params.Param.Measurement.Type-link.html